Core/Graphics/Source/FrameBufferManager.h

```diff
@@ -33,5 +35,6 @@ namespace Babylon
         std::list<FrameBuffer> m_frameBuffers{};
 
         FrameBuffer m_default;
+        FrameBuffer m_backBuffer;
```

Comment:
Only minor thing left is the naming of these. It's a bit unclear what these two frame buffers mean now. It would be good to add some comments and make the naming consistent between the variable and the getter function.